pkg/gui/controllers/helpers/refs_helper.go

```diff
@@ -587,3 +581,19 @@ func (self *RefsHelper) ParseRemoteBranchName(fullBranchName string) (string, st
 func IsSwitchBranchUncommittedChangesError(err error) bool {
 	return strings.Contains(err.Error(), "Please commit your changes or stash them before you switch branch")
 }
+
+func (self *RefsHelper) GetSuggestedBranchName(suggestedBranchName string) (string, error) {
+	var err error
+	if suggestedBranchName != "" {
+		return suggestedBranchName, nil
+	}
+
+	suggestedBranchName, err = utils.ResolveTemplate(self.c.UserConfig().Git.BranchPrefix, nil, template.FuncMap{
+		"runCommand": self.c.Git().Custom.TemplateFunctionRunCommand,
+	})
+	if err != nil {
+		return suggestedBranchName, err
+	}
+	suggestedBranchName = strings.ReplaceAll(suggestedBranchName, "\t", " ")
+	return suggestedBranchName, err
```

Comment:
Since err is always nil here, I'd find it easier to read if we return nil. See fixup a3b2007b4cf0.